plugins/modules/site_workflow_manager.py

```diff
@@ -125,9 +125,10 @@
                 description: |
                     Floor number within the building site (e.g., 5). This value can only be specified during the creation of the
                     floor and cannot be modified afterward.
+                    This field is mandatory in 2.3.7.6 version.
                 type: int
               units_of_measure:
-                description: The unit of measurement for floor dimensions, typically 'feet' or 'meters'.
+                description: The unit of measurement for floor dimensions, typically 'feet' or 'meters'. This field is introduced in 2.3.7.6 version.
```

Comment:
Specifies the unit of measurement for floor dimensions, typically 'feet' or 'meters'. This field is introduced from version 2.3.7.6 onwards.

Reply:
Modified